### PR TITLE
Add type defintions

### PR DESCRIPTION
**Pull request by hisuwh**

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+
+export interface TruncateProps {
+    lines?: number;
+    ellipsis?: React.ReactNode | ((element: React.ReactNode) => React.ReactNode);
```
Comment (patrik-piskay) [inline index.d.ts]: this could be `string`, `ReactNode` or `function`. looks like `string` is not covered here?

File: index.d.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    lineHeight?: number;
```
Comment (patrik-piskay) [inline index.d.ts]: this could be `number` or `string`

File: index.d.ts (hunk 1 above)
Comment (hisuwh) [reply index.d.ts]: ReactNode covers string

Comment (hisuwh) [reply index.d.ts]: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L148

**Comment by patrik-piskay** [comment]
Thanks again 👍

**Pull request merged**